department-of-veterans-affairs/abd-vro
# Mcp 1618 - Use shared objects to remove the redundant codes

Sharing common data objects to remove the redundant codes.

## What was the problem?
Some data objects used can be shared among the projects and the redundant code can be removed.

Associated tickets or Slack threads:
MCP-1618

## How does this fix it?[^1]
1. Shared some data objects in shared:domain and svc-lighthouse-api, and removed some data objects in svc-lighthouse-api.
2. Shared some data objects in shared:domain and mock-bip-claim-api and removed the redundant codes. - reverted.

## How to test this PR
- Step 1. Build the app
- Step 2. Check anything broken.


[^1]: [Pull-Requests guidelines](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Pull-Requests). If PR is significant, update [Current Software State](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Current-Software-State) wiki page.
[^secrel]: To check if a PR will succeed in the SecRel workflow, [test PRs in the SecRel pipeline](https://github.com/department-of-veterans-affairs/abd-vro-internal/wiki/Secure-Release-process#to-test-prs-in-the-secrel-pipeline).
